samples/android/app/src/main/res/layout/activity_anoncreds.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is essentially a duplicate 5 times. Have one file and change in the code what needs to be chabged.